src/Client/Web/Bit.Client.Web.BlazorUI/Components/Pivot/BitPivot.razor.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public partial class BitPivot
+    {
+        protected override string RootElementClass => "bit-pvt";
+
+        private string selectedKey;
+        private OverflowBehavior _overflowBehavior = OverflowBehavior.None;
+        private LinkFormat _linkFormat = LinkFormat.Links;
+        private LinkSize _linkSize = LinkSize.Normal;
+
+        [Parameter]
+        public string DefaultSelectedKey { get; set; } = "0";
+
+        [Parameter]
+        public RenderFragment? ChildContent { get; set; }
+
+        [Parameter]
+        public OverflowBehavior OverflowBehavior
+        {
+            get => _overflowBehavior;
+            set
+            {
+                _overflowBehavior = value;
+                ClassBuilder.Reset();
+            }
+        }
+
+        [Parameter]
+        public LinkFormat LinkFormat
+        {
+            get => _linkFormat;
+            set
+            {
+                _linkFormat = value;
+                ClassBuilder.Reset();
+            }
+        }
+
+        [Parameter]
+        public LinkSize LinkSize
+        {
+            get => _linkSize;
+            set
+            {
+                _linkSize = value;
+                ClassBuilder.Reset();
+            }
+        }
+
+        [Parameter]
+        public bool HeadersOnly { get; set; } = false;
+
+
+        [Parameter]
+        public EventCallback<BitPivotItem> OnLinkClick { get; set; }
+
+        [Parameter]
+        public string SelectedKey
+        {
+            get => selectedKey;
+            set
+            {
+                if (value == selectedKey) return;
+                selectedKey = value;
+                _ = SelectedKeyChanged.InvokeAsync(value);
+            }
+        }
+
+        [Parameter]
+        public EventCallback<string> SelectedKeyChanged { get; set; }
+
+        public IDictionary<string, BitPivotItem> Items = new Dictionary<string, BitPivotItem>();
```

Comment:
Is this property required to be public?

Reply:
changed to internal because Items is used by an instance of Pivot in BitPivotItem